common/djangoapps/nodebb/tasks.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tasks to synchronize users with NodeBB
+"""
+from logging import getLogger
+
+from requests.exceptions import ConnectionError
+from celery.task import task
+
+from common.lib.nodebb_client.client import NodeBBClient
+
+log = getLogger(__name__)
+
+RETRY_DELAY = 20 # seconds
+
+@task(bind=True, default_retry_delay=RETRY_DELAY, max_retries=None)
+def task_create_user_on_nodebb(self, username, **kwargs):
+    """
+    Celery task to create user on NodeBB
+    """
+    try:
+        NodeBBClient().users.create(username=username, kwargs=kwargs)
+        print 'Success: User creation task for user: {}'.format(username)
```

Comment:
please use logs instead of print statements

Reply:
Used.